install-TSM.md

```diff
@@ -0,0 +1,61 @@
+# Installing TAP on clusters onboarded to Tanzu Service Mesh
+
+This topic describes the workaround for using Tanzu Service Mesh with TAP. You cannot install TAP on a cluster that has Tanzu Service Mesh attached. If you want to install TAP on a cluster where Tanzu Service Mesh is attached, follow the procedure below.
+
+This workaround describes how Tanzu Service Mesh can be configured to ignore the TAP namespaces. This allows to install TAP, while Tanzu Service Mesh continues to satisfy other connectivity concerns.
```

Comment:
Should be "This allows you to install TAP"

Reply:
Fixed